src/client.ts

```diff
@@ -452,6 +477,84 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
     return this._client.getFeatureVariableDouble(feature, variable, user.id, user.attributes);
   }
 
+  /**
+   * Returns value for the given json variable attached to the given feature
+   * flag
+   * @param {string} feature
+   * @param {string} variable
+   * @param {string} [overrideUserId]
+   * @param {optimizely.UserAttributes} [overrideAttributes]
+   * @returns {(string | null)}
+   * @memberof OptimizelyReactSDKClient
+   */
+  public getFeatureVariableJson(
+    feature: string,
+    variable: string,
+    overrideUserId?: string,
+    overrideAttributes?: optimizely.UserAttributes
+  ): unknown {
+    const user = this.getUserContextWithOverrides(overrideUserId, overrideAttributes);
+    if (user.id === null) {
+      return null
+    }
+    return this._client.getFeatureVariableJson(
+      feature,
+      variable,
+      user.id,
+      user.attributes,
+    );
+  }
+
+  /**
+   * Returns dynamically-typed value of the variable attached to the given
+   * feature flag. Returns null if the feature key or variable key is invalid.
+   * @param {string} featureKey
+   * @param {string} variableKey
+   * @param {string} [overrideUserId]
+   * @param {optimizely.UserAttributes} [overrideAttributes]
+   */
+  getFeatureVariable(
+      featureKey: string,
+      variableKey: string,
+      overrideUserId: string,
+      overrideAttributes?: optimizely.UserAttributes
+  ): unknown {
+    const user = this.getUserContextWithOverrides(overrideUserId, overrideAttributes);
+    if (user.id === null) {
+      return null
+    }
+    return this._client.getFeatureVariable(
+        featureKey,
+        variableKey,
+        user.id,
+        user.attributes,
+    );
+  }
+
+  /**
+   * Returns values for all the variables attached to the given feature flag
+   * @param {string} featureKey
+   * @param {string} overrideUserId
+   * @param {optimizely.UserAttributes} [overrideAttributes]
+   * @returns {(object | null)}
```

Comment:
can we mention the correct `{ [variableKey: string]: unknown }` return type here too